protected/humhub/modules/space/views/space/_layout.php

```diff
@@ -4,47 +4,54 @@
  * @var string $content
  */
 
+use humhub\modules\space\widgets\Header;
+use humhub\modules\space\widgets\Menu;
+use humhub\modules\space\widgets\SpaceContent;
+use humhub\modules\space\widgets\Sidebar;
+use humhub\modules\activity\widgets\Stream;
+use humhub\modules\space\modules\manage\widgets\PendingApprovals;
+use humhub\modules\space\widgets\Members;
 use humhub\widgets\FooterMenu;
 
 $space = $this->context->contentContainer;
 ?>
+
 <div class="container space-layout-container">
     <div class="row">
         <div class="col-md-12">
-            <?php echo humhub\modules\space\widgets\Header::widget(['space' => $space]); ?>
-
+            <?= Header::widget(['space' => $space]); ?>
         </div>
     </div>
     <div class="row space-content">
         <div class="col-md-2 layout-nav-container">
-            <?php echo \humhub\modules\space\widgets\Menu::widget(['space' => $space]); ?>
+            <?= Menu::widget(['space' => $space]); ?>
             <br>
         </div>
 
         <?php if (isset($this->context->hideSidebar) && $this->context->hideSidebar) : ?>
             <div class="col-md-10 layout-content-container">
-                <?= \humhub\modules\space\widgets\SpaceContent::widget([
+                <?= SpaceContent::widget([
                     'contentContainer' => $space,
                     'content' => $content
-                ]) ?>
+                ]);
+                ?>
                 <?= FooterMenu::widget(['location' => FooterMenu::LOCATION_FULL_PAGE]); ?>
             </div>
-        <?php else: ?>
+        <?php else : ?>
             <div class="col-md-7 layout-content-container">
-                <?= \humhub\modules\space\widgets\SpaceContent::widget([
+                <?= SpaceContent::widget([
                     'contentContainer' => $space,
                     'content' => $content
-                ]) ?>
+                ]);
+                ?>
             </div>
             <div class="col-md-3 layout-sidebar-container">
-                <?php
-                echo \humhub\modules\space\widgets\Sidebar::widget(['space' => $space, 'widgets' => [
-                        [\humhub\modules\activity\widgets\Stream::className(), ['streamAction' => '/space/space/stream', 'contentContainer' => $space], ['sortOrder' => 10]],
-                        [\humhub\modules\space\modules\manage\widgets\PendingApprovals::className(), ['space' => $space], ['sortOrder' => 20]],
-                        [\humhub\modules\space\widgets\Members::className(), ['space' => $space], ['sortOrder' => 30]]
+                <?= Sidebar::widget(['space' => $space, 'widgets' => [
+                        [PendingApprovals::className(), ['space' => $space], ['sortOrder' => 10]],
```

Comment:
Please use ::class instead of className(). ClassName() is depcreated and will be removed with yii 2.1/ yii 3